docs/user/openstack/deploy_baremetal_workers.md

```diff
@@ -184,10 +185,14 @@ where the bare-metal machines will be attached.
           oc create -v baremetalMachineSet.yaml
 
 
+## Known issues
+
+Bare metal nodes are not supported on Kuryr installations.
```

Comment:
"on clusters that use Kuryr," if you like.

Reply:
Thanks!